.travis.yml

```diff
@@ -2,4 +2,12 @@ language: java
 install: ""
 script: TERM=dumb ./gradlew clean check
 jdk:
-    - openjdk7
+    - oraclejdk8
```

Comment:
What do you mean? For the byte code generation? JDK 1.8 is for test runtime only. See commit message for 9415a2f0ccd4cff6641d8fe15d9f72e60fd0fba6